### ISSUE TITLE
LdaMallet.show_topics(formatted=False) fails 

### ISSUE THREAD

**Issue by frederik-elwert** [RaRe-Technologies/gensim#1150]
Seems to be another side effect of the topn/num_topics issue:

In `gensim/models/wrappers/ldamallet.py`

```python
            if formatted:
                topic = self.print_topic(i, topn=num_words)
            else:
                topic = self.show_topic(i, topn=num_words)
```

should be

```python
            if formatted:
                topic = self.print_topic(i, topn=num_words)
            else:
                topic = self.show_topic(i, num_words=num_words)
```

**Comment by piskvorky** [comment]
@tmylk why the param name inconsistency?

**Comment by markroxor** [comment]
Everything seems alright here https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/wrappers/ldamallet.py#L237

**Comment by tmylk** [comment]
Closing issue as fixed in #1066.

@piskvorky There is an inconsistency between `show_topic`-`topn` and `show_topics`-`num_words` in `LdaModel` which is replicated along other topic models in order to standardise the api.

`show_topics` and `print_topics` were standardised among topic models in #755 to match `LdaModel` to use `num_words` as the argument names were confusing. 
Later #771 changed `show_topic` and `print_topic` to use `num_words` just in `ldamallet`. That was a way to standardise it with `show_topics`, but in retrospect it was not the best move.
Then `print_topic` was changed back to use `topn` like all the other models do when ldamallet inherited from `BaseTopicModel`. Though `show_topic` has remained - that is the inconsistency you see above.

Raised an issue #1198

**Comment by piskvorky** [comment]
Alright, makes sense, thanks. Let's make it consistent, definitely.
